x-pack/plugin/sql/qa/server/src/main/resources/agg.csv-spec

```diff
@@ -1325,3 +1325,386 @@ F              |1964-10-18T00:00:00.000Z|1952-04-19T00:00:00.000Z
 M              |1965-01-03T00:00:00.000Z|1952-02-27T00:00:00.000Z
 ;
 
+
```

Comment:
Please, add one more test, that uses two SUMs and some other elements: `SELECT SUM(bytes_in), SUM(bytes_out), client_ip, COUNT(*) AS c FROM logs WHERE client_ip = '10.0.0.0/16' AND NVL(bytes_out, 0) = 0 GROUP BY client_ip ORDER BY c DESC, SUM(bytes_in) ASC NULLS FIRST`